src/SFA.DAS.PR.Application/SFA.DAS.PR.Application.csproj

```diff
@@ -16,6 +16,7 @@
 	<ItemGroup>
 		<PackageReference Include="Azure.Identity" Version="1.11.2" />
 		<PackageReference Include="Microsoft.Identity.Client" Version="4.60.3" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
We're trying to not use newtonsoft, use System.Text.Json instead.  It's not difficult to swap, I\m happy to pair with you to get it done quickly